### PR TITLE
Decrease size of widget pagination

### PR DESCRIPTION
**Pull request by ranbena**
- [x] Feature

## Description
A continuation #4101 to make the now fixed pagination take up less space in widgets.
Using [ant's mini pagination](https://ant.design/components/pagination/#components-pagination-demo-mini) with `size="small"`.
Most of the PR is propagating the context `query` / `widget`.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Before|After
------|------
Screen Shot 2019-09-09 at 10 40 08 |  Screen Shot 2019-09-09 at 10 40 05

### CODE REVIEW

File: client/app/visualizations/table/Renderer.jsx [hunk 1]
```diff
@@ -59,14 +59,15 @@ export default function Renderer({ options, data }) {
   }
 
   return (
-    <div className="table-visualization-container">
+    <div className="table-visualization-container" data-context={context}>
```
Comment (kravets-levko) [inline client/app/visualizations/table/Renderer.jsx]: Why is this needed?

Comment (ranbena) [reply client/app/visualizations/table/Renderer.jsx]: I knew I forgot sth...

**Comment by ranbena** [comment]
@kravets-levko I'm not in love with the code here. Lmk if you got a better idea how to let the renderer decide between pagination sizes.

**Comment by arikfr** [comment]
Need to rebase this because 4101 was already merged.

**Comment by ranbena** [comment]
Done.
Also updated the tests.
